client/client.go

```diff
@@ -15,88 +15,85 @@
 package client
 
 import (
-	"fmt"
-	"net/http"
-	"net/url"
+	"log"
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/client/restclient"
-	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/client/unversioned"
 )
 
-type AppControllerClient struct {
-	*http.Client
-	Root                   *url.URL
-	dependenciesURL        *url.URL
-	resourceDefinitionsURL *url.URL
+type Interface interface {
```

Comment:
Full name for it is k8s-AppController.client.Interface, and eventually i hope it becomes appcontroller.client.Interface. I don't really see the need to add 'AppController' prefix here. At least until there come more public interfaces in the package.